ProjFS.Mac/PrjFSKextTests/KauthHandlerTests.mm

```diff
@@ -282,47 +282,55 @@ - (void)testShouldHandleVnodeOpEvent {
     XCTAssertEqual(kauthResult, KAUTH_RESULT_DEFER);
 }
 
-- (void)testCurrentProcessWasSpawnedByRegularUser {
+- (void)testCurrentProcessIsAllowedToHydrate {
     // Defaults should pass for all tests
-    XCTAssertTrue(CurrentProcessWasSpawnedByRegularUser());
+    XCTAssertTrue(CurrentProcessIsAllowedToHydrate());
     MockProcess_Reset();
 
-    // Process is a service user and does not have a parent
+    // Process is not a 'amfid' service user and does not have a parent
```

Comment:
The comment here was previously wrong (all processes have parents), so if we're going to edit it, may as well get it right. Perhaps something like:

```
// Process and its parents are owned by a system user, and the executable is not a whitelisted service.
```